requests_toolbelt/adapters/appengine.py

```diff
@@ -59,6 +79,24 @@ def init_poolmanager(self, connections, maxsize, block=False):
         self.poolmanager = _AppEnginePoolManager(self._validate_certificate)
 
 
+class InsecureAppEngineAdapter(AppEngineAdapter):
+    """A variant of the the transport adapter for Requests to use urllib3's GAE
```

Comment:
I know we don't make pydocstyle a required check in Travis, but please try to follow PEP-0257. For example,

```
An always-insecure GAE adapter for Requests.

This is a variant of the AppEngineAdapter provided by this module that will **never** validate certificates by default. **Use with caution!**
```